docs/user/alerting/rule-types/es-query.asciidoc

```diff
@@ -221,6 +226,30 @@ Labels:
 // NOTCONSOLE
 --
 
+`context.link`::
+(string) The URL for the rule that generated the alert.
+For example: `/app/management/insightsAndAlerting/triggersActions/rule/47754354-d894-49d3-87ec-05745a74e2b7`.
+
+`context.message`::
+(string) A preconstructed message for the rule.
+For example: +
+`Document count is 100 in the last 1h. Alert when greater than 50.`
+
+`context.sourceFields`::
+(array of strings)
+// TBD: When and what type of information is returned here?
```

Comment:
Looks like these were added in this PR: https://github.com/elastic/kibana/pull/171129 and contains a list of user entered source fields, if populated.

Reply:
This is the format:

```
sourceFields: {
  'host.hostname': ['host-1'],
  'host.id': ['1'],
  'host.name': ['host-1'],
},
```

so an object where each key is a source field that the user identified and the values are an array of strings values found for that field.

Reply:
Thanks! Updated in 2ad9423664fd1c5b5175def62283046b3702e4f4